packages/react/src/lib/__tests__/default.spec.tsx

```diff
@@ -0,0 +1,106 @@
+import renderer from 'react-test-renderer';
+import { SlateToReact } from './../serializers';
+import { config as defaultReactConfig } from './../config/default';
+
+describe('React conversion', () => {
+  test('convert domhandler element to React element', async () => {
+    const slate = [
+      {
+        children: [
+          {
+            text: 'Paragraph',
+          },
+        ],
+        type: 'p',
+      },
+    ];
+
+    const tree = renderer.create(<SlateToReact node={slate} />).toJSON();
+    expect(tree).toMatchInlineSnapshot(`
+      <p>
+        Paragraph
+      </p>
+    `);
+  });
```

Comment:
Inline snapshots are much easier to review. Might be a good idea to replace the combined fixture approach with this approach - it can help simplify fixture files in `@slate-serializers/tests` and make test results easier to review, especially since these fixtures were moved to `@slate-serializers/tests` so that they can be shared in tests across different packages.